ABCE_XML_BACKUPS/abce_Custom_Sampler.xml

```diff
@@ -0,0 +1,115 @@
+<?xml version="1.0" ?>
```

Comment:
The files in ABCE_XML_BACKUPS and OUT should be removed.

Reply:
Agreed. I had added them to the .gitignore but they still got added to the commit.